### PR TITLE
Vishal/add staking faqs

### PR DESCRIPTION
**Pull request by vishalchangrani**
Closes: #4984 (https://github.com/dapperlabs/flow-go/issues/4984)

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: docs/content/intro/flow-token.mdx [hunk 1]
```diff
@@ -165,7 +165,7 @@ If a node operator has delegators, they cannot withdraw their own tokens
 such that their own staked tokens would fall below the minimum requirement for that node type.
 This is enforced at the protocol level.
 
-- Collector Nodes: 250,000 FLOW
+- Collection Nodes: 250,000 FLOW
 - Consensus Nodes: 500,000 FLOW
```
Comment (joshuahannan) [inline docs/content/intro/flow-token.mdx:169]:
```suggestion
- Consensus Nodes: 250,000 FLOW
```

File: docs/content/node-operation/node-roles.mdx [hunk 2]
```diff
@@ -30,43 +30,43 @@ Unlike most blockchains, not all Flow nodes are equal. Flow nodes all specialize
...
+Consensus nodes act as checkpoints against other Collection nodes. They are responsible for checking that a critical number of Collection nodes reviewed and signed for the transaction. Collection nodes are held accountable by Consensus nodes. A common concern with proof-of-work- and proof-of-stake based systems is that a small subset of the population of nodes can control important resources such as the mining or stake needed to produce and vote on blocks, which is a degradation of the security of the system. By lowering the requirements to participate, Flow makes it extremely difficult and expensive to coordinate a Byzantine majority of Consensus nodes.
```
Comment (joshuahannan) [inline docs/content/node-operation/node-roles.mdx:42]:
```suggestion
Consensus nodes act as checkpoints against Collection nodes. They are responsible for checking that a critical number of Collection nodes reviewed and signed for the transaction. Collection nodes are held accountable by Consensus nodes. A common concern with proof-of-work- and proof-of-stake based systems is that a small subset of the population of nodes can control important resources such as the mining or stake needed to produce and vote on blocks, which is a degradation of the security of the system. By lowering the requirements to participate, Flow makes it extremely difficult and expensive to coordinate a Byzantine majority of Consensus nodes.
```
I don't think the "other" is necessary here

File: docs/content/node-operation/node-roles.mdx [hunk 3]
```diff
@@ -30,43 +30,43 @@ Unlike most blockchains, not all Flow nodes are equal. Flow nodes all specialize
...
+Consensus nodes have minimal bandwidth and computation requirements, allowing even a modest computing device (any consumer-grade hardware) to participate in the voting process and ensure the safety of the network. Many networks claim open participation, yet substantial resources — stake, computation, or otherwise — are needed to partake. Maintaining such barriers to entry undermines the security of the network. Lowering the participation requirements preserves the security of the network by providing a high degree of byzantine fault tolerance since it becomes exceedingly difficult for a subset of bad actors to subvert the network.
 
-Consensus Nodes are required to stake a minimum of 500,000 FLOW to be a confirmed node operator.
+Consensus nodes are required to stake a minimum of 500,000 FLOW to be a confirmed node operator.
```
Comment (joshuahannan) [inline docs/content/node-operation/node-roles.mdx:46]:
```suggestion
Consensus nodes are required to stake a minimum of 500,000 FLOW to be a confirmed node operator. This requirement is relaxed to 250,000 FLOW at network launch, but will be increased later.
```

File: docs/content/staking/faq.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+### Will staking rewards be automatically re-staked?
+
+There will be *no* automatic re-staking of staking rewards with Flow Port (ledger/blocto).
+
+If you staked using a staking provider such as Kraken, Blocto or Finoa, they should be able to provide you more details.
+
+### Can a validator change its fees? Would the Account Key contract need to stay in Beta permanently to enable the validator to change the fee percentage? Does this change take effect immediately or at the next epoch?
+
+For the beta period we’re enforcing a minimum fee of 10%.
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:21]:
```suggestion
For the beta period we’re enforcing a minimum delegation fee of 8%.
```

File: docs/content/staking/faq.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+In the future it will be a parameter you can update.
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:22]:
```suggestion
```
The current understanding is that they won't be able to update their fee

File: docs/content/staking/faq.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+The contract includes a function you can call to update if you would like but this wouldn’t require the contract to be in beta.
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:23]:
```suggestion
```

File: docs/content/staking/faq.mdx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+Rewards are only paid out every epoch, so the fee at the time of the epoch payout would be the one applied.
+
+### Can a token holder stake to multiple nodes? If yes, how is the stake split between them?
+
+A token holder will be able to stake multiple nodes in the future.
+However, currently there is a restriction in place such that there is a 1-to-1 relationship between an account address and an active stake.
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:29]:
```suggestion
However, currently there is a restriction in place from the token lockup rules such that there is a 1-to-1 relationship between an account address and an active stake.
```

**Review by joshuahannan — APPROVED** [review]
Looks great! I just have a few comments

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Can we remove the other pages that aren;t the FAQ. Those pages got major updated in another PR (including some of the corrections Josh made here)

**Comment by vishalchangrani** [comment]
> Can we remove the other pages that aren't the FAQ. Those pages got major updated in another PR (including some of the corrections Josh made here)

sorry about that

**Review by caseyturnerarrington — APPROVED** [review]

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**